usaspending_api/search/v2/views/spending_by_award.py

```diff
@@ -109,6 +110,20 @@ def validate_request_data(request_data):
         models = [
             {"name": "fields", "key": "fields", "type": "array", "array_type": "text", "text_type": "search", "min": 1},
             {"name": "subawards", "key": "subawards", "type": "boolean", "default": False},
+            {
```

Comment:
Not a blocker. Since these filters are identically defined in two files, they could be moved into `usaspending-api/usaspending_api/common/validator/award_filter.py` as a separate dictionary from `AWARD_FILTER` (e.g. `LEGACY_AWARD_FILTER`)

Up to you if that is done.

Reply:
I actually did this (with the same variable name), then thought better of it since the other "extra" fields above weren't done that way. I'm also indifferent to this one.